test/correctness/simd_op_check.cpp

```diff
@@ -1909,6 +1937,10 @@ int main(int argc, char **argv) {
         test.set_num_threads(1);
     }
 
+    if (getenv("FILTER")) {
```

Comment:
nit: is this really an improvement over the optional command-line arg we already have?

Reply:
Err, I forgot to remove this. But yes, it is, I add it locally every time I'm doing something with simd_op_check. You can't pass a command line arg to the makefile. It's a few extra steps to work around, and running simd_op_check drops a lot of files wherever you run it from.

Reply:
Perhaps simd_op_check shouldn't do that. I've had it bomb my working directory in that way too, and it's super annoying. It should just make a tmp subdir (and print what it is so you can go inspect the files if you really want to)

Reply:
+100, output to /tmp dir always would be much better.

re: "FILTER", can we at least make it something slightly less generically named?